source/nodejs/adaptivecards/src/serialization.ts

```diff
@@ -729,13 +737,15 @@ export abstract class SerializableObject {
 
     protected populateSchema(schema: SerializableObjectSchema) {
         let ctor = <any>this.constructor;
+        let properties: PropertyDefinition[] = [];
 
         for (let propertyName in ctor) {
             try {
                 let propertyValue = ctor[propertyName];
 
                 if (propertyValue instanceof PropertyDefinition) {
-                    schema.add(propertyValue);
+                    // schema.add(propertyValue);
```

Comment:
nit: probably don't need to keep commented code

Reply:
Indeed - removed.